app/controllers/EditionsController.scala

```diff
@@ -190,7 +191,7 @@ class EditionsController(db: EditionsDB,
   def publishIssue(id: String, version: EditionIssueVersionId) = EditEditionsAuthAction { req =>
     val lastProofedIssueVersion = db.getLastProofedIssueVersion(id)
     // Protect against stale requests.
-    if (lastProofedIssueVersion.filter(_.equals(version)).isEmpty) {
+    if (version != Publishing.ProofingNotRequiredMagicVersion && !lastProofedIssueVersion.exists(_.equals(version))) {
```

Comment:
The issue knows whether it supports proofing, or not (`issue.supportsProofing`?) – could we fetch the issue and use that to make this decision, rather than relying on a magic version?

If so, we could do the same thing instead of the magic version check in Publishing.scala.

Reply:
Done 😁